maker/src/actor_system.rs

```diff
@@ -146,7 +161,7 @@ where
         let endpoint_listen = daemon::libp2p_utils::create_listen_tcp_multiaddr(&libp2p_socket)
             .expect("to parse properly");
 
-        let (supervisor, _listener_actor) = supervisor::Actor::with_policy(
+        let (supervisor, listener_actor) = supervisor::Actor::with_policy(
```

Comment:
🔧 Can you do this in a separate commit? Would be nice to explain the fix separately.

Reply:
done!